Subject: Enable CHANMUTE optimization (for xpp, mostly)
Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Forwarded: no

Enable the OPTIMIZE_CHANMUTE code: don't waste CPU time on channels
whose audio is not useful. Currently only used by xpp drivers, and
disabled by default by upstream.

--- a/include/dahdi/dahdi_config.h
+++ b/include/dahdi/dahdi_config.h
@@ -176,7 +176,7 @@
 /*
  * Skip processing PCM if low-level driver won't use it anyway
  */
-/* #define	OPTIMIZE_CHANMUTE */
+#define	OPTIMIZE_CHANMUTE
 
 
 /*
